tools/ci-ts/tests/flux-monitor.test.ts

```diff
@@ -1,96 +1,156 @@
-import { FluxAggregatorFactory } from '@chainlink/contracts/ethers/v0.6/FluxAggregatorFactory'
-import { contract, helpers as h, matchers } from '@chainlink/test-helpers'
+import 'isomorphic-unfetch'
 import { assert } from 'chai'
 import { ethers } from 'ethers'
-import 'isomorphic-unfetch'
-import { JobSpec } from '../../../operator_ui/@types/operator_ui'
-import fluxMonitorJob from '../fixtures/flux-monitor-job'
-import * as clClient from '../test-helpers/chainlink-cli'
+import { cloneDeep } from 'lodash'
+import { FluxAggregatorFactory } from '@chainlink/contracts/ethers/v0.6/FluxAggregatorFactory'
+import { contract, helpers as h, matchers } from '@chainlink/test-helpers'
+import ChainlinkClient from '../test-helpers/chainlinkClient'
+import _fluxMonitorJob from '../fixtures/flux-monitor-job'
 import {
```

Comment:
Would prefer a namespace import when the destructuring gets this big. `import * as t from '../test-helpers/common'`